batch/batch/front_end/front_end.py

```diff
@@ -609,8 +650,15 @@ async def get_job_container_log(request, batch_id):
     containers = job_tasks_from_spec(record)
     if container not in containers:
         raise web.HTTPBadRequest(reason=f'unknown container {container}')
-    job_log = await _get_job_container_log(app, batch_id, job_id, container, record)
-    return web.Response(body=job_log)
+    container_log = await _get_job_container_log(app, batch_id, job_id, container, record)
+
+    resp = web.StreamResponse()
+    await resp.prepare(request)
+    if isinstance(container_log, ReadableStream):
```

Comment:
```suggestion
    if container_log is not None:
```
right?

Reply:
Oop ya, this is cruft from a prior implementation where I was sometimes also returning `bytes`.